rust-toolchain.toml

```diff
@@ -0,0 +1,10 @@
+[toolchain]
+# Keep in sync with Cargo.toml's `rust-version`.
+channel = "1.61.0"
+components = [
+  "cargo",
+  "clippy",
+  "rust-std",
+  "rustc",
+  "rustfmt",
+]
```

Comment:
Out of curiosity, is there any way to specify some sort of "dev-only" components?  We also want to use `llvm-feature-preview` (and `cargo install grcov`) during full coverage testing, but we don't need them in regular builds.

Reply:
No, this isn't possible. But per https://rust-lang.github.io/rustup/concepts/components.html, you can add a component to an already existing toolchain with `rustup component add rust-docs`.

But we could proactively add `llvm-tools-preview` here. The only downside is a slightly slower install time for people who won't use it, including our CI jobs that won't use it.

Reply:
Ah, no worries then - `rustup component add` is already what we do (and it's only in CI anyway), I was more curious if there was a way to formalise a "dev" group in the toolchain file.  Thanks!